src/app/payment/paymentMethod/MonerisPaymentMethod.tsx

```diff
@@ -19,10 +22,21 @@ const MonerisPaymentMethod: FunctionComponent<MonerisPaymentMethodProps> = ({
         },
     }), [containerId, initializePayment]);
 
+    const validateInstrument = (_shouldShowNumberField: boolean, selectedInstrument?: CardInstrument): React.ReactNode => {
+        const trustedShippingAddress = selectedInstrument && selectedInstrument.trustedShippingAddress;
+
+        return (<CreditCardValidation
```

Comment:
I think for Hosted providers we are about to use Hosted Fields for Verification because of security 

We implemented it on Mollie

https://github.com/bigcommerce/checkout-js/blob/master/src/app/payment/paymentMethod/MolliePaymentMethod.tsx#L88